### PR TITLE
Added device.js module for getting device info

### PR DESCRIPTION
**Pull request by amanelis**
Jira: https://charge.atlassian.net/browse/PROF-451

### CODE REVIEW

File: src/device.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * 
+ */
+export const DeviceMiddleware = (req, res, next) => {
+	device.getInfo(req.headers['user-agent'], req.ip, (err, res) => {
+    req.origin = res
+    req.origin.client = {
+      x_forwarded_for: req.headers['X-Forwarded-For'],
+      x_forwarded_proto: req.headers['X-Forwarded-Proto'],
+      x_forwarded_port: req.headers['X-Forwarded-Port'],
+    }
```
Comment (dmytro-kulchytskyi) [inline src/device.js:25]: maybe use camelcase field naming style here(without underscores) to follow style of other our code

Comment (amanelis) [reply src/device.js:25]: I did snake case only because I see all log messages use that formatting.

Comment (dmytro-kulchytskyi) [reply src/device.js:25]: @amanelis oh, ok, then no problem :)

Comment (amanelis) [reply src/device.js:25]: nvm, you are right. Will fix.

Comment (dmytro-kulchytskyi) [reply src/device.js:25]: as you wish, if its a standard it's ok

**Review by dmytro-kulchytskyi — APPROVED** [review]

File: src/device.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import deviceIPLocation from 'device-ip-location'
```
Comment (NFhbar) [inline src/device.js:1]: This function belongs in `interface` as an export

**Review by NFhbar — CHANGES_REQUESTED** [review]

**Pull request closed without merging**